Editor/Scripts/RGActionInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RegressionGames
+{
+    [System.Serializable]
+    public class RGActionInfo
+    {
+        public string Object;
```

Comment:
Can we add some comments here? Wasn't sure what Object was.

Reply:
Added 👍